frontend-react/src/components/GlobalNav/GlobalNav.tsx

```diff
@@ -1,6 +1,6 @@
 import { PlusOutlined } from '@ant-design/icons';
 import { CodeResponse, useGoogleLogin } from '@react-oauth/google';
-import { Button, Divider, Select, Space, Tabs as AntdTabs, Typography } from 'antd';
+import { Button, Divider, notification, Select, Space, Tabs as AntdTabs, Typography } from 'antd';
```

Comment:
Why is the `notification` only non-capitalized import?

Reply:
Because it's not a component. It's a method and that's how Antd defined their exports so I don't have a choice